addons/core/functions/fnc_getVehicleRadios.sqf

```diff
@@ -0,0 +1,36 @@
+#include "script_component.hpp"
+
+/*
+    Name: TFAR_fnc_getVehicleRadios
+
+    Author(s):
+        Jaffa
+
+    Description:
+        Gets a list of radios in the specified vehicle
+
+    Parameters:
+        0: Object - vehicle
+
+    Returns:
+         Array of LR radio arrays in format: 
+             0 - Object - Vehicle, 1 - String - Radio Settings ID
+         or empty array if no vehicle radios found
+
+    Example:
+        _radios = _vehicle call TFAR_fnc_getVehicleRadios;
+*/
+params ["_vehicle"];
```

Comment:
I think, you could add a default value and a required datatype since this function is considered as API/Public stuff:
`params [["_vehicle", objNull, [objNull]]];`